seatunnel-e2e/seatunnel-spark-e2e/src/test/java/org/apache/seatunnel/e2e/spark/SparkContainer.java

```diff
@@ -84,6 +85,8 @@ public Container.ExecResult executeSeaTunnelSparkJob(String confFile) throws IOE
         master.copyFileToContainer(MountableFile.forHostPath(confPath), targetConfInContainer);
 
         // TODO: use start-seatunnel-spark.sh to run the spark job. Need to modified the SparkStarter can find the seatunnel-core-spark.jar.
+        String jar = SPARK_JAR_PATH.replaceAll("\\\\", "/");
+        String conf = targetConfInContainer.replaceAll("\\\\", "/");
```

Comment:
Please add a comment, why you need to do a replacement here, e.g. `we need to replase the \\ due to campatible with Windows`.